### PR TITLE
hipchat support

### LINKED ISSUE #203: Add HipChat Support
Add Support for HipChat

### PR DESCRIPTION
**Pull request by kazuminn**
## What did you implement:

Closes #203

## Todos:
You don't have to satisfy all of the following.

- [ ] Write tests
- [x] Write documentation
- [x] Check that there aren't other open pull requests for the same issue/feature
- [x] Format your source code by `make fmt`
- [x] Pass the test by `make test`
- [x] Provide verification config / commands
- [x] Enable "Allow edits from maintainers" for this PR
- [x] Update the messages below

***Is this ready for review?:*** NO  
***Is it a breaking change?:*** NO

### CODE REVIEW

File: report/hipchat.go [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+)
+
+// HipChatWriter send report to HipChat
+type HipChatWriter struct{}
+
+func (w HipChatWriter) Write(rs ...models.ScanResult) (err error) {
+	conf := config.Conf.HipChat
+	room := conf.Room
+	token := conf.AuthToken
+	pp.Print()
```
Comment (kotakanbe) [inline report/hipchat.go]: Remove this line if for debug

File: report/hipchat.go [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+			message = `<a href="https://nvd.nist.gov/vuln/detail\">` +  vinfo.CveID + "</a>" + "<br/>" + strconv.FormatFloat(maxCvss.Value.Score, 'f', 1, 64) + " " + "(" + severity + ")" + "<br/>" + vinfo.Summaries(config.Conf.Lang, r.Family)[0].Value
+
+			err = postMessage(room, token, message)
+			if err != nil {
+				return err
+			}
+		}
+
+	}
+	return err
```
Comment (kotakanbe) [inline report/hipchat.go]: return nil if no err occurred.

File: report/hipchat.go [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+	"net/url"
+	"strings"
+	"strconv"
+)
+
+// HipChatWriter send report to HipChat
+type HipChatWriter struct{}
+
+func (w HipChatWriter) Write(rs ...models.ScanResult) (err error) {
+	conf := config.Conf.HipChat
```
Comment (kotakanbe) [inline report/hipchat.go]: I think these local variables (conf, room and token) have no meaning.
Use config.Conf.conf, room, token directory.

**Comment by kotakanbe** [comment]
Would you please update the document?

https://vuls.io/docs/en/usage-report.html
https://github.com/vulsdoc/vuls/blob/master/docs/usage-report.md

**Comment by kazuminn** [comment]
👍

**Comment by kazuminn** [comment]
Done document writing.

**Pull request merged**